data/lang/module-secondhand/en.json

```diff
@@ -87,6 +87,14 @@
     "description": "Regarding installing a component on the ship",
     "message": "How do I fit it?"
   },
+  "NO_LONGER_AVAILABLE_0": {
+    "description": "If the sentence is hard to translate, just make up an excuse why the item is no longer available.",
+    "message": "Sorry, it has just been sold!"
+  },
+  "NO_LONGER_AVAILABLE_1": {
+    "description": "If the sentence is hard to translate, just make up an excuse why the item is no longer available.",
+    "message": "It seem like my colleague just sold it to another customer. Sorry!"
```

Comment:
Should it be "It seems like"?

Reply:
Yes. "He, she, it - 's' commts mit", as the Germans are taught in English-class. (Since us poor Scandinavians don't distinguished between have/has, we tend to mess those up far more common than we realize, myself included)